build_tools/python_deploy/build_linux_packages.sh

```diff
@@ -129,6 +129,7 @@ function build_iree_runtime() {
 function build_iree_runtime_instrumented() {
   IREE_HAL_DRIVER_CUDA=ON IREE_BUILD_TRACY=ON IREE_ENABLE_RUNTIME_TRACING=ON \
   IREE_RUNTIME_CUSTOM_PACKAGE_SUFFIX="-instrumented" \
+  IREE_HAL_VULKAN_FEATURE_ENABLE_DEBUG_UTILS=2 \
```

Comment:
this should be `ON`

Reply:
Okay

Reply:
Changed the value to 2 
based on this 
https://github.com/nod-ai/SHARK-Runtime/blob/shark/runtime/src/iree/hal/drivers/vulkan/api.h#L39

will change it to ON